pulp_python/app/serializers.py

```diff
@@ -214,6 +215,23 @@ class Meta:
         model = python_models.PythonPackageContent
 
 
+class PythonOneShotUploadSerializer(serializers.Serializer):
+    """
+    A Serializer for PythonOneShotUpload.
+    """
+
+    repository = serializers.HyperlinkedRelatedField(
+        help_text=_('A URI of the repository.'),
+        required=False,
+        queryset=Repository.objects.all(),
+        view_name='repositories-detail',
+    )
+    file = serializers.FileField(
+        help_text=_("The python file."),
```

Comment:
Might be helpful to give a bit more detail here. ie, a whl or whatever-- this sounds like mything.py could be valid.